app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizardStateProvider.js

```diff
@@ -1,31 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { bindMethods } from 'patternfly-react';
 
 class ModalWizardStateProvider extends React.Component {
   constructor() {
     super();
     this.state = { activeStepIndex: 0 };
```

Comment:
should we get rid of the constructor here and use the class properties syntax for the state?

Reply:
sure...happy to move to the `state = {}` syntax too... just a few!

Reply:
updated... i think this syntax is cleaner...except when you are using props to initialize state.